build-logic/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt

```diff
@@ -308,6 +310,17 @@ val Project.isExperimentalTestFilteringEnabled: Boolean
     get() = systemProperty(TEST_FILTERING_ENABLED).getOrElse("false").toBoolean()
 
 
+val Project.predictiveTestSelectionEnabled: Boolean
+    get() = systemProperty(PREDICTIVE_TEST_SELECTION_ENABLED)
+        .map { it.toBoolean() }
+        .orElse(provider {
+            val protectedBranches = listOf("master", "release")
```

Comment:
Shouldn't it always be disabled for `master`/`release` at least on `CI`?

Reply:
It is unless someone explicitly enables it via `-D...`.